### PR TITLE
add main scaffold

### PR DESCRIPTION
**Pull request by pabloolveira**
add main scaffold

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def foo():
+    print("bar bar")
```
Comment (SergiA07) [inline main.py]: I think this should be justy "bar"

File: main.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+foo()
```
Comment (SergiA07) [inline main.py:4]: you should invoke the function later

**Review by SergiA07 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (pabloolveira) [reply main.py]: done

File: main.py (hunk 2 above)
Comment (pabloolveira) [reply main.py:4]: I don't think so

Comment (SergiA07) [reply main.py:4]: ok, never mind

**Review by SergiA07 — APPROVED** [review]

**Pull request merged**